test/parallel/test-http-default-port.js

```diff
@@ -31,7 +32,7 @@ const assert = require('assert');
 const hostExpect = 'localhost';
 const fs = require('fs');
 const path = require('path');
-const fixtures = path.join(common.fixturesDir, 'keys');
+const fixtures = path.join(fixturesModule.fixturesDir, 'keys');
```

Comment:
This can be a `fixures.path('keys')`

Reply:
I think a better change would be to update the `fs.readFileSync()` calls a few lines later to use the fixture keys call.

Reply:
For instance...

```js
const options = {
  key: fixtures.readSync('agent1-key.pem'),
  cert: fixtures.readSync('agent1-cert.pem')
};
```